LEGO1/mxaudiomanager.h

```diff
@@ -0,0 +1,21 @@
+#ifndef MXAUDIOMANAGER_H
+#define MXAUDIOMANAGER_H
+
+#include "mxmediamanager.h"
+
+// VTABLE 0x100dc6e0
+class MxAudioManager : public MxMediaManager
+{
+public:
+  MxAudioManager();
+  virtual ~MxAudioManager() override;
+
+  void LockedReinitialize(MxS8);
+  void Reinitialize();
+protected:
+  void Init();
+
+  int m_unk2c;
```

Comment:
While we don't have any code that actually uses it or idea of its purpose it would be good to declare it as `undefined4` type